modules/text2vec-openai/clients/vectorizer.go

```diff
@@ -16,6 +16,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	vectorizer2 "github.com/weaviate/weaviate/modules/text2vec-openai/vectorizer"
```

Comment:
Need to sort imports for linter to pass 🙂

Reply:
Sorry missed that. 

The other failing presubmit told me to gen-code-from-swagger.sh but that didn't seem to change any files. 

Here's the tail of the output
```
✅ already up to date: entities/search/ref.go
✅ already up to date: entities/search/result.go
✅ already up to date: entities/search/select_property.go
✅ already up to date: entities/verbosity/verbosity.go
✅ already up to date: entities/schema/test_utils/nested_properties.go
✅ already up to date: entities/schema/crossref/crossref_source_test.go
✅ already up to date: entities/schema/crossref/bulk_builder.go
✅ already up to date: entities/schema/crossref/crossref_test.go
✅ already up to date: entities/schema/crossref/bulk_builder_test.go
✅ already up to date: entities/schema/crossref/crossref_source.go
✅ already up to date: entities/schema/crossref/crossref.go
✅ already up to date: entities/schema/accessors.go
✅ already up to date: entities/schema/data_types_test.go
✅ already up to date: entities/schema/data_types.go
✅ already up to date: entities/schema/backward_compat_test.go
✅ already up to date: entities/schema/inverted_index_config.go
✅ already up to date: entities/schema/vector_index_config.go
✅ already up to date: entities/schema/nested_properties_test.go
✅ already up to date: entities/schema/accessors_test.go
✅ already up to date: entities/schema/nested_properties.go
✅ already up to date: entities/schema/schema.go
✅ already up to date: entities/schema/validation.go
✅ already up to date: entities/schema/backward_compat.go
✅ already up to date: entities/schema/multi_tenancy.go
✅ already up to date: entities/schema/validation_test.go
✅ already up to date: entities/schema/errors.go
✅ already up to date: entities/lsmkv/errors.go
✅ already up to date: entities/deepcopy/models_deepcopy.go
✅ already up to date: entities/vectorindex/flat/config.go
✅ already up to date: entities/vectorindex/flat/config_test.go
✅ already up to date: entities/vectorindex/hnsw/config.go
✅ already up to date: entities/vectorindex/hnsw/bq_config.go
✅ already up to date: entities/vectorindex/hnsw/pq_config.go
✅ already up to date: entities/vectorindex/hnsw/config_test.go
✅ already up to date: entities/vectorindex/common/config.go
✅ already up to date: entities/vectorindex/config.go
✅ already up to date: entities/replication/config.go
✅ already up to date: entities/errorcompounder/compounder_thread_safe.go
✅ already up to date: entities/errorcompounder/compounder.go
✅ already up to date: entities/additional/classification.go
✅ already up to date: entities/additional/distance.go
✅ already up to date: entities/additional/group.go
✅ already up to date: entities/additional/replication.go
✅ already up to date: entities/errors/errors_graphql.go
✅ already up to date: entities/errors/errors_http.go
✅ already up to date: entities/errors/errors_remote_client.go
✅ already up to date: entities/modulecapabilities/classification.go
✅ already up to date: entities/modulecapabilities/config.go
✅ already up to date: entities/modulecapabilities/graphql.go
✅ already up to date: entities/modulecapabilities/backup.go
✅ already up to date: entities/modulecapabilities/vectorizer.go
✅ already up to date: entities/modulecapabilities/texttransformer.go
✅ already up to date: entities/modulecapabilities/additional.go
✅ already up to date: entities/modulecapabilities/client.go
✅ already up to date: entities/modulecapabilities/searcher.go
✅ already up to date: entities/modulecapabilities/module.go
✅ already up to date: entities/backup/descriptor.go
✅ already up to date: entities/backup/descriptor_test.go
✅ already up to date: entities/backup/status.go
✅ already up to date: entities/backup/errors.go
✅ already up to date: entities/cyclemanager/interval.go
✅ already up to date: entities/cyclemanager/cyclemanager.go
✅ already up to date: entities/cyclemanager/cyclecallbackgroup.go
✅ already up to date: entities/cyclemanager/cyclecallbackgroup_test.go
✅ already up to date: entities/cyclemanager/cyclecallbackctrl_test.go
✅ already up to date: entities/cyclemanager/ticker_test.go
✅ already up to date: entities/cyclemanager/cyclecallbackctrl.go
✅ already up to date: entities/cyclemanager/cyclemanager_test.go
✅ already up to date: entities/cyclemanager/ticker.go
✅ already up to date: entities/cyclemanager/errors.go
✅ already up to date: entities/storobj/buffer_pool.go
✅ already up to date: entities/storobj/storage_object_test.go
✅ already up to date: entities/storobj/storage_object.go
✅ already up to date: entities/storobj/enrich_schema_datatypes.go
✅ already up to date: entities/storobj/parse_single_object.go
✅ already up to date: entities/storobj/errors.go
✅ already up to date: deprecations/gen.go
🖋️  successfully created header: deprecations/data.go
✅ already up to date: deprecations/main.go
Success
```